appinventor/components/src/com/google/appinventor/components/runtime/AppInventorCompatActivity.java

```diff
@@ -55,8 +61,13 @@
   private static Theme currentTheme = Theme.PACKAGED;
   private static int primaryColor;
   private AppCompatDelegate appCompatDelegate;
+  private RelativeLayout relativeLayout;
+  com.google.android.material.floatingactionbutton.FloatingActionButton floatingActionButton;
```

Comment:
I'd rather that the FAB be created by the component itself and its logic managed there so that if people don't want it in the project it doesn't crash the app. Further, having the FAB on older versions of Android causes the companion to crash so its creation needs to be keyed to the earliest version of Android supported by the library, which is SDK 14 (Ice Cream Sandwich).